src/main/java/mate/web/filter/AuthenticationFilter.java

```diff
@@ -0,0 +1,48 @@
+package mate.web.filter;
+
+import java.io.IOException;
+import javax.servlet.Filter;
+import javax.servlet.FilterChain;
+import javax.servlet.FilterConfig;
+import javax.servlet.ServletException;
+import javax.servlet.ServletRequest;
+import javax.servlet.ServletResponse;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.service.DriverService;
+
+public class AuthenticationFilter implements Filter {
+    private static final String DRIVER_ID = "driver_id";
+    private static final Injector injector = Injector.getInstance("mate");
+    private DriverService driverService = (DriverService) injector
+            .getInstance(DriverService.class);
+
+    @Override
+    public void init(FilterConfig filterConfig) throws ServletException {
+
+    }
+
+    @Override
+    public void doFilter(ServletRequest servletRequest, ServletResponse servletResponse,
+                         FilterChain filterChain) throws IOException, ServletException {
+        HttpServletRequest req = (HttpServletRequest) servletRequest;
+        HttpServletResponse resp = (HttpServletResponse) servletResponse;
+        Long driverId = (Long) req.getSession().getAttribute(DRIVER_ID);
+        String url = req.getServletPath();
+        if (url.equals("/login") || url.equals("/drivers/create")) {
```

Comment:
wrote about this in the chat

Reply:
sorry, I was think you mean I shouldn't forget about point what mentioned in this chat

Reply:
ok, next time i will use 'channel', thanks)